### DISCUSSION TITLE
Is it possible to remove the brand from the login page?

### DISCUSSION THREAD

**Discussion by yablochnyi** [filamentphp/filament#6300] [Help]
/*
    |--------------------------------------------------------------------------
    | Brand Name
    |--------------------------------------------------------------------------
    |
    | This will be displayed on the login page and in the sidebar's header.
    |
    */

    'brand' => 'Test',

**Comment by danharrin** [comment]
You can extend the Login page class, override `hasLogo()` to return false, and then register that new class in the config file.

**Reply by ArtMin96** [reply]
It is only available in v3. In v2, you should extend Login page class and override the `render` method
